### DISCUSSION TITLE
Testnet USDC from https://staging.aave.com/faucet/ not shown in METAMASK

### DISCUSSION THREAD

**Discussion by UdayQxf2** [openbuildxyz/solidity_bootcamp#102] [Q&A]
I am getting Testnet USDC from https://staging.aave.com/faucet/ not showing on Metamask

I approved the Transaction in Metamask.

It is showing on Aave page at USDC Testnet Faucet but not showing on Metamask.

I require to send USDC from Metamask to my Smartcontract                                   

results of SEPOLIA ETHERSCAN

Transaction Hash: 0x2aa782b2acb0c24d0e9ca2a5b72c0b1fe25686b31031ca3c953319f3821570fb 0xD62f1B97...3cEe3D67f on 0xC959483D...EDEb3f42D From: 0xD62f1B978D60a91e34AB133DF68b33B3cEe3D67f Interacted With (To): 0xC959483DBa39aa9E78757139af0e9a2EDEb3f42D 

ERC-20 Tokens Transferred: 0x00000000...000000000 To 0xD62f1B97...3cEe3D67f For 10,000 USDC (USDC) Value: 0 ETH ($0.00) 

Transaction Fee: 0.001117175946134094 ETH ($0.00) Gas Price: 24.444258498 Gwei (0.000000024444258498 ETH)

thanks 
Uday

**Comment by fanofmusk** [comment]
您的邮件已收到，谢谢
